R/bundle.R

```diff
@@ -924,6 +932,11 @@ createAppManifest <- function(appDir, appMode, contentCategory, hasParameters,
   # add metadata
   manifest$metadata <- metadata
 
+  # if there is a target image, attach it to the environment
+  if (!is.null(image)) {
+    manifest$environment$image <- image
```

Comment:
Do you need something like the following to make sure that `manifest$environment` exists?

```r
if (!is.null(image)) {
    manifest$environment <- list(image = image)
}
```